test/z_dateformat_test.erl

```diff
@@ -0,0 +1,23 @@
+%% @author Marc Worrell <marc@worrell.nl>
+
+-module(z_dateformat_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+
+dateformat_year_y_test() ->
+    ?assertEqual(<<"-0999">>,  z_dateformat:format({{  -999,1,1},{0,0,0}}, "Y", [])),
+    ?assertEqual(<<"0999">>,   z_dateformat:format({{   999,1,1},{0,0,0}}, "Y", [])),
```

Comment:
Unwanted behaviour.

Reply:
What is unwanted about this?

Reply:
See above: you don’t want to render 0313 in templates.